content/blog/2018/istio-authorization/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Micro-Segmentation with Istio Authorization
+description: Describe Istio authorization characterics and how it can be used for different use cases.
```

Comment:
Describes Istio's authorization features and how to use them in various use cases.